src/server/models/images.py

```diff
@@ -34,15 +34,17 @@
 from src.server.models import ArtifactLink
 from src.server.helper import ARCH_X86_64, ARCH_ARM64
 
+
 class V2ImageRecord:
     """ The ImageRecord object """
 
     # pylint: disable=W0622
-    def __init__(self, name, link=None, id=None, created=None, arch=ARCH_X86_64):
+    def __init__(self, name, link=None, id=None, created=None, arch=ARCH_X86_64, metadata=None):
         # Supplied
         self.name = name
         self.link = link
-        
+        self.metadata = metadata if metadata else {}
```

Comment:
Usually this is done by just setting the default value above to what you want the default value to be:
arch=ARCH_X86_64, metadata={}):

Then this line is just:
self.metadata = metadata